docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -32,7 +33,79 @@ policy that describes the {fleet-server} configuration. In large scale
 self-managed deployments or on {ess-product}[hosted {ess}] on {ecloud},
 {fleet-server} is typically run as a dedicated {agent} communication host, but
 you can optionally use it for data collection on self-managed clusters. For more
-details, refer to <<fleet-server-deployment>>.
+details, refer to <<fleet-server-scalability>>.
+
+[discrete]
+[[deployment-models]]
+== {fleet-server} deployment models
+
+Administrators deploying the {agent} have a few deployment choices
+available to satisfy their organization's requirements. {fleet-server} can be
+deployed:
+
+//Nima: Should we say something about hosting on ECE here?
```

Comment:
ECE is covered in the cloud portion, as in fleet server deployment is not going to be different.